doc/manual/src/advanced-topics/distributed-builds.md

```diff
@@ -38,11 +38,11 @@ contains Nix.
 
 > **Warning**
 >
-> If you are building via the Nix daemon, it is the Nix daemon user
-> account (that is, `root`) that should have SSH access to the remote
-> machine. If you can’t or don’t want to configure `root` to be able to
-> access to remote machine, you can use a private Nix store instead by
-> passing e.g. `--store ~/my-nix`.
+> If you are building via the Nix daemon, it is the Nix daemon user account
+> (that is, `root`) that should have SSH access to a certain user on the remote
```

Comment:
```suggestion
> (that is, `root`) that should have SSH access to a user on the remote
```

If you say "certain", I understand it may be a particular one. If that doesn't matter, keep it "a user". If we can provide more information (such as a trusted user), add that (and add links to relevant documentation, such as the trusted users setting).

Also, please let's use one sentence per line in markdown, because reflows produce diff noise that is hard to parse and review.

Reply:
> If you say "certain", I understand it may be a particular one.

You are correct.

> If that doesn't matter, keep it "a user".

I only wanted to emphasize that it doesn't need to be necessarily root on the remote machine.

> If we can provide more information (such as a trusted user), add that (and add links to relevant documentation, such as the trusted users setting).

According to my experience the remote user doesn't need to be trusted.

> Also, please let's use one sentence per line in markdown (even if it's not consistent everywhere yet), because reflows produce diff noise that is hard to parse and review.

I won't mind - pushed a fix now. The diff noise is reduced, but the line breaks look a bit random now.